### PR TITLE
CLOUD-2186 Upgrade actions-python to use ubuntu-latest

### PR DESCRIPTION
**Pull request by vibindaniel**
# Description

Created actions out of the scripts that were previously used here.
Refactored the scripts, but most of the logic is the same.

Fixes [#2186](https://usxpress.atlassian.net/browse/CLOUD-2186)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change
  (fix or feature that would cause existing
  functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested in demo-python-flash-api by running through actions-python
https://github.com/variant-inc/demo-python-flask-variant-api/actions/runs/4939725042/jobs/8831073524

- [x] Sanity Testing

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added documentation to test

### CODE REVIEW

File: create-release-tag/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+<!-- action-docs-outputs -->
+
+<!-- action-docs-runs -->
+## Runs
+
+This action is a `composite` action.
+<!-- action-docs-runs -->
+
+## Usage
```
Comment (zumic96) [inline create-release-tag/README.md:30]: Fix this section.

File: build-push-image/ecr_create.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "repositoryName": "$ECR_REPOSITORY"
+    "imageTagMutability": "MUTABLE"
+  }
+}
+EOF
+    )
+    echo "::debug::$(curl -sSfL -X POST \
+      "https://$SECRET__LAZY_API_URL/tenants/apps/profiles/production/regions/$AWS_REGION/ecr/repo" \
+      -d "$data")"
+    echo "::notice::ECR Repository $ECR_REPOSITORY already created"
```
Comment (zumic96) [inline build-push-image/ecr_create.sh:32]: Remove "already".

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      - master
+      - main
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Create Tag
+        uses: actions/github-script@v5
```
Comment (zumic96) [inline .github/workflows/release.yml:17]: Can local create-release-tag action be used here?

File: build-push-image/build-push-core/action.yml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+  required env:
+    IMAGE_VERSION
+inputs:
+  dockerfile_dir_path:
+    description: Directory path to the dockerfile
+    required: false
+    default: .
+  ecr_registry:
+    description: ECR repository name
```
Comment (zumic96) [inline build-push-image/build-push-core/action.yml:15]: Update description.

File: build-push-image/build-push-core/README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+<!-- action-docs-description -->
+
+<!-- markdownlint-disable line-length -->
+<!-- action-docs-inputs -->
+## Inputs
+
+| parameter | description | required | default |
+| --- | --- | --- | --- |
+| dockerfile_dir_path | Directory path to the dockerfile | `false` | . |
+| ecr_registry | ECR repository name | `true` |  |
```
Comment (zumic96) [inline build-push-image/build-push-core/README.md:18]: Update description.

**Review by zumic96 — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -1,21 +1,25 @@
 # Description
 
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+Please include a summary of the change and which issue is fixed.
+Please also include relevant motivation and context.
+List any dependencies that are required for this change.
 
-Fixes [#8](https://usxtech.atlassian.net/browse/CLOUD-8)
+Fixes [#8](https://usxpress.atlassian.net/browse/CLOUD-8)
```
Comment (cameran20) [inline .github/pull_request_template.md:7]: I don't have access to this link

Comment (vibindaniel) [reply .github/pull_request_template.md:7]: Because it doesn't exist and is a dummy.

**Review by cameran20 — APPROVED** [review]

**Pull request merged**